linz-data-importer/linz_data_importer.py

```diff
@@ -44,13 +44,12 @@
 from .gui.Service_dialog import ServiceDialog
 
 # Hardcoded service .see #20 for enhancement
-SER=['',
-    'geodata.nzdf.mil.nz',
+SER=['basemaps.linz.govt.nz',
```

Comment:
There was an empty string entry in the original list - is that needed for something?